File: code-samples/messaging/global-business/get-sync-dlr.cs
using System;
using com.Messente.Api.Api;
using com.Messente.Api.Client;

namespace Example
{
    public class GetSyncDlr
    {
        public static void Main()
        {
            Configuration conf = new Configuration
            {
                Username = "YOUR_MESSENTE_API_USERNAME",
                Password = "YOUR_MESSENTE_API_PASSWORD"
            };

            var apiInstance = new DeliveryReportApi(conf);

            try
            {
                var result = apiInstance.RetrieveDeliveryReport("<omnimessage_id>");
                Console.WriteLine(result.ToJson());
            }
            catch (Exception e)
            {
                Console.WriteLine("Exception when retrieving delivery report: " + e.Message);
            }
        }
    }
}

File: code-samples/messaging/global-business/get-sync-dlr.php
<?php
require_once(__DIR__ . '/../vendor/autoload.php');

use Messente\Api\Api\DeliveryReportApi;
use Messente\Api\Configuration;

$config = Configuration::getDefaultConfiguration()
    ->setUsername('YOUR_MESSENTE_API_USERNAME')
    ->setPassword('YOUR_MESSENTE_API_PASSWORD');

$apiInstance = new DeliveryReportApi(new GuzzleHttp\Client(), $config);

try {
    $result = $apiInstance->retrieveDeliveryReport('<omnimessage_id>');
    print_r($result);
} catch (Exception $e) {
    echo 'Exception when retrieving delivery report: ', $e->getMessage(), PHP_EOL;
}
